### DISCUSSION TITLE
Is it possible to run a OOP background task in full trust just like the app?

### DISCUSSION THREAD

**Discussion by abhi-msft** [microsoft/WindowsAppSDK#1078] [Q&A]
I have a Project Reunion 0.8.1 desktop app (full trust) with an out-of-process background task from CsWinRT component.

Is it possible to configure and run this task in full trust to perform few activities like copying files etc?

**Comment by AdamBraden** [comment]
1.	For Inproc, your full trust app can implement IBackgroundTask and use COM activation. See: [Create and register a winmain COM background task](https://docs.microsoft.com/en-us/windows/uwp/launch-resume/create-and-register-a-winmain-background-task )
2.	For OOP, you can create an OOP WinRT exe component and register it for background activation.

@andreww-msft is looking into this for future AppLifecycle Activation scenarios.

**Reply by abhi-msft** [reply]
Corrected my question. I am looking to run OOP with full trust.
